### ISSUE TITLE
Purchase issue

### ISSUE THREAD

**Issue by crashem** [LoukaMB/SynapseX#201]
So, I was recommended to buy synapse by a friend and then as I proceed to buy it. It says wait 24-72 hours so then I wait then I wait still nothing. I mean I payed with a steam card about 20$. It’s been a week now basically and I’m still waiting... 
  
Sincerely, 
Crasheem
  
Email: markogaldamez11@gmail.com

**Comment by finnpatrick** [comment]
Yeah, same with me too. I hope the Synapse team redeems our codes soon.
